### PR TITLE
Feature/cli_description

### PR DESCRIPTION
**Pull request by Markuu-s**
### Description of the Change
Added description to a commands and groups. Added show of position arguments.

### CODE REVIEW

File: core/cli/node/main.cpp [hunk 1]
```diff
@@ -9,56 +9,124 @@
...
-          CMD("delete", Node_wallet_delete),
-          {GROUP("market")({
-              CMD("add", Node_wallet_market_add),
+      {GROUP("wallet", "Manage wallet")({
+          CMD("new",
+              Node_wallet_new,
+              "Generate a new key of the given type",
+              {"type(bls|secp256k1 (default: secp256k1))"}),
+          CMD("list", Node_wallet_list, "List wallet address"),
+          CMD("add-balance", Node_wallet_addBalance, ""),
```
Comment (wer1st) [inline core/cli/node/main.cpp:94]: Why without a description?

File: core/cli/node/main.cpp [hunk 2]
```diff
@@ -9,56 +9,124 @@
...
+              Node_wallet_verify,
+              "Verify the signature of a message",
+              "signing address",
+              "hexMessage",
+              "signature"),
+          CMD("delete",
+              Node_wallet_delete,
+              "Soft delete an address from the wallet - hard deletion "
+              "needed "
+              "for permanent removal",
```
Comment (wer1st) [inline core/cli/node/main.cpp:120]:
```suggestion
              "needed for permanent removal",
```

File: core/cli/run.hpp [hunk 3]
```diff
@@ -93,7 +93,15 @@ namespace fc::cli {
           }
         }
       }
-      fmt::print("name:\n  {}\n", fmt::join(cmds, " "));
+      fmt::print("name:\n  {}", fmt::join(cmds, " "));
```
Comment (turuslan) [inline core/cli/run.hpp:96]:
```suggestion
      fmt::print("usage:\n  {}", fmt::join(cmds, " "));
```

File: core/cli/run.hpp [hunk 4]
```diff
@@ -93,7 +93,15 @@ namespace fc::cli {
...
+      for(const auto &arg : tree->argusage) {
+        fmt::print("  <{}>", arg);
+      }
+      fmt::print("\n");
+
+      if (not tree->description.empty()) {
+        fmt::print("usage:\n  {}\n", tree->description);
```
Comment (turuslan) [inline core/cli/run.hpp:103]:
```suggestion
        fmt::print("description:\n  {}\n", tree->description);
```

**Review by turuslan — APPROVED** [review]

File: core/cli/node/filplus.hpp [hunk 5]
```diff
@@ -138,21 +138,21 @@ namespace fc::cli::cli_node {
       auto state =
           cliTry(getCbor<VerifiedRegistryActorStatePtr>(ipfs, actor.head));
 
-      auto encoded_params1 = cliTry(codec::cbor::encode(
+      auto encoded_params = cliTry(codec::cbor::encode(
           vm::actor::builtin::v0::verified_registry::AddVerifier::Params{
               *args.from, *args.amount
           }));
-      SignedMessage signed_message1 = cliTry(api->MpoolPushMessage(
+      const SignedMessage signed_message1 = cliTry(api->MpoolPushMessage(
```
[the remark this answers is not in the document]
Comment (Markuu-s) [reply core/cli/node/filplus.hpp:145]: Fixed

File: core/cli/node/main.cpp (hunk 1 above)
Comment (Markuu-s) [reply core/cli/node/main.cpp:94]: Fixed

File: core/cli/run.hpp (hunk 4 above)
Comment (Markuu-s) [reply core/cli/run.hpp]: fixed

File: core/cli/node/main.cpp (hunk 2 above)
Comment (Markuu-s) [reply core/cli/node/main.cpp]: Fixed

**Review by wer1st — APPROVED** [review]

**Pull request merged**